GX-TimeFrequency/src/minimax_grids.F90

```diff
@@ -0,0 +1,462 @@
+!  Copyright (C) 2020-2022 Green-X library
```

Comment:
unify your headers. It think a !********************... line is missing

Reply:
Fixed